Tests/Invoke-DockerTag.Tests.ps1

```diff
@@ -57,14 +52,87 @@ Describe 'Tag docker images' {
 
         It 'tags docker image with image name with target tag' {
 
-            Mock -CommandName "Invoke-Command" $code -Verifiable -ModuleName $script:moduleName
-
             Invoke-DockerTag -SourceImage 'lalaland' -TargetImage 'lololand' -TargetTag 'target'
-
             Assert-MockCalled -CommandName "Invoke-Command" -ModuleName $script:moduleName
             $result = GetMockValue -Key "mock"
             Write-Debug $result
             $result | Should -BeLikeExactly "docker tag lalaland:latest lololand:target"
         }
     }
+
+    Context 'Docker tags private registry images' {
+
+        It 'tags private docker image as public docker image with image name only' {
+
+            Invoke-DockerTag -SourceRegistry 'artifactoryfqdn' -SourceImage 'lalaland' -TargetImage 'lololand'
+            Assert-MockCalled -CommandName "Invoke-Command" -ModuleName $script:moduleName
+            $result = GetMockValue -Key "mock"
+            Write-Debug $result
+            $result | Should -BeLikeExactly "docker tag artifactoryfqdn/lalaland:latest lololand:latest"
+        }
+
+        It 'tags private docker image as public docker image with image name and tag' {
+
```

Comment:
Consider getting rid of these empty lines.

Reply:
I'll remove it. But it's much easier to read for me when there are some extra line breaks.